indexing-service/src/main/java/org/apache/druid/indexing/common/task/NoopTask.java

```diff
@@ -159,17 +143,8 @@ public void stopGracefully(TaskConfig taskConfig)
   @Override
   public TaskStatus runTask(TaskToolbox toolbox) throws Exception
   {
-    if (firehoseFactory != null) {
-      log.info("Connecting firehose");
-    }
-    try (Firehose firehose = firehoseFactory != null ? firehoseFactory.connect(null, null) : null) {
-
-      log.info("Running noop task[%s]", getId());
-      log.info("Sleeping for %,d millis.", runTime);
```

Comment:
Could this logging be useful for understanding test runs better?

Reply:
No, I don't see it adding much value. These logs fill up the logs and make it difficult to find important things such as warnings and errors.